pkg/timezone/timezone_unix.go

```diff
@@ -0,0 +1,25 @@
+//go:build darwin || linux
+
+package timezone
+
+import (
+	"fmt"
+	"os"
+	"runtime"
+	"strings"
+)
+
+// Timezone returns the local timezone if found. If not, an error is returned.
+func Timezone() (string, error) {
+	tz, err := os.Readlink("/etc/localtime")
```

Comment:
According to [tzlocal/unix.py](https://github.com/regebro/tzlocal/blob/d5fb741969269c1cc0f4c1a87a66c10426863761/tzlocal/unix.py) it's slightly more complicated than just reading `/etc/localtime` depending on the flavor of Unix/Linux. Let's try to match the logic in tzinfo?